operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,19 @@
+# Audit Logs
+
+How to enable and disable audit logging for host and Kubernetes APIs.
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional and will generate some load and data on the non-compute nodes \(NCNs\).
+Host and Kubernetes API audit logging is not enabled by default.
+
+To enable or disable host and Kubernetes API audit logging, change the CSI setting:
+
+* `ncn-mgmt-node-auditing-enabled`: Set to `true` to enable host logging or to `false` to disable host logging.
+* `k8s-api-auditing-enabled`: Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+When the CSI setting is changed, each NCN will need to be restarted to apply the new setting.
+
+Host audit logs are stored in the `/var/log/audit/HostOS` directory on each NCN. Host audit logging uses a maximum of 60GB on each NCN when using log rotation settings. The log rotation settings are enabled when the Ansible playbook is run.
```

Comment:
I believe that the reference to the Ansible playbook probably should be taken out..  I think that refers back to the way we used to apply the settings (before we switched it to a csi setting).   I think we may just have to test it out, but I'd assume that it would need to be modified to say "The log rotation settings are enabled after editing the csi settings and rebooting the NCNs."  I am guessing this may come down to having someone test these instructions.